package.json

```diff
@@ -7,19 +7,19 @@
     "ensure-env": "node ./tools/scripts/ensure-env.js",
     "lint": "echo 'Warning: TODO - Define Linting.'",
     "pretest": "npm-run-all -s lint bootstrap",
+    "seed": "node seed/seedChallenges",
     "test": "npm-run-all -p test:*",
     "test:client": "cd ./client && npm test && cd ../",
     "test:curriculum": "echo 'Warning: TODO - Define Testing.'",
     "test:server": "echo 'Warning: TODO - Define Testing.'",
+    "test:tools": "cd ./tools/challenge-md-parser && npm test && cd ../scripts/seed && npm test && cd ../../../",
```

Comment:
This is messy, but can be fixed by moving devDeps to the root. At the minute it would require having another `@babel/core` installed at the root of the project